src/components/SearchTagDisplay.jsx

```diff
@@ -26,7 +26,10 @@ export default function SearchTagDisplay({ tags, removeTag }) {
             }
             type={
               ChipTypeStyles[
-                stringUtils.toUpperEveryWord(tag.label).replaceAll(" ", "")
+                stringUtils
```

Comment:
We're styling some of the chips based on the filter groups (ex: we style based on topic, not topic types) and all keys are stored as lowecase strings, so we do need it here.

We could change the underlying ChipTypeStyle to be lowercase for the filter groups (basically just make topic lowercase) but that might cause issues elsewhere plus it would make the styling enum inconsistent.

Reply:
I removed the string utils method, but we still need to remove the spaces and numbers in order get the correct enum values.